src/gui/gui.pri

```diff
@@ -119,6 +120,7 @@ SOURCES += \
     $$PWD/transferlistsortmodel.cpp \
     $$PWD/transferlistwidget.cpp \
     $$PWD/updownratiodialog.cpp \
+    $$PWD/uithememanager.cpp \
```

Comment:
@jagannatharjun:  It is still in the wrong place.  These filenames should be listed in **alphabetical order**.  

In the English version of the Latin alphabet, `i` comes before `p`.  

```
$$PWD/updownratiodialog.cpp \
$$PWD/uithememanager.cpp \
```

…should be… 

```
$$PWD/uithememanager.cpp \
$$PWD/updownratiodialog.cpp \
```